tools/auto_fill_stable_diffusion_cache.py

```diff
@@ -0,0 +1,192 @@
+# coding=utf-8
```

Comment:
The only difference with the auto_fill_inference_cache script seem to be the input parameters: would it be possible to just extend the existing script ?

Reply:
Sounds good, will do.